pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -635,6 +635,11 @@ type ServiceAccountConfig struct {
 	// with a validity duration of this value.
 	// +optional
 	MaxTokenExpiration *metav1.Duration `json:"maxTokenExpiration,omitempty" protobuf:"bytes,4,opt,name=maxTokenExpiration"`
+	// AcceptedIssuers is an additional set of issuers that are used to determine which service account tokens are accepted.
+	// These values are not used to generate new service account tokens. Only useful when service account tokens are also
+	// issued by another external system or a change of the current issuer that is used for generating tokens is being performed.
```

Comment:
Nit: We could also specify in the field doc string (that then will be part of the API reference doc) that it is allowed to specify this field only with K8s > 1.22.